index.js

```diff
@@ -115,7 +115,13 @@ class SideMenu extends Component {
    */
   handlePanResponderMove(e: Object, gestureState: Object) {
     if (this.state.left.__getValue() * this.menuPositionMultiplier() >= 0) {
-      this.state.left.setValue(this.prevLeft + gestureState.dx);
+      let newLeft = this.prevLeft + gestureState.dx;
+
+      if (!this.props.bounceBackOnOverdraw && Math.abs(newLeft) > this.props.openMenuOffset) {
```

Comment:
I already gave you the reasoning - none of them was `because I can`. :P The current way side menu works is with `bounceBackOnOverdraw` set to true. That's how it's been working since the first day, that's how Basic demo example works, that's how all my production apps look like.

Now, we are adding ability to turn that off for those who were complaining (in #154) they want to prevent it. Two choices, based entirely on app UX. If you decide to make it default w/o ability to configure, I will have to start using another solution and we don't know if I am going to be the only one. It is important to make it a non-breaking update instead of next major bump.

What else you would like to discuss here? ;)